scrapy_zyte_api/handler.py

```diff
@@ -25,28 +26,30 @@ def __init__(
         self, settings: Settings, crawler: Crawler, client: AsyncClient = None
     ):
         super().__init__(settings=settings, crawler=crawler)
-        self._client: AsyncClient = client if client else AsyncClient()
+        if not client:
+            try:
+                client = AsyncClient(
+                    api_key=settings.get('ZYTE_API_KEY'),
+                    api_url=settings.get('ZYTE_API_URL') or API_URL,
+                    n_conn=settings.getint('CONCURRENT_REQUESTS'),
+                )
+            except NoApiKey:
+                logger.warning(
+                    "'ZYTE_API_KEY' must be set in the spider settings or env var "
+                    "in order for ScrapyZyteAPIDownloadHandler to work."
+                )
+                raise NotConfigured
+        self._client: AsyncClient = client
+        logger.info(f"Using Zyte API Key: {self._client.api_key[:7]}")
```

Comment:
Would it make sense to include something like ..., to indicate that's not the full key?